beacon-chain/core/helpers/committee.go

```diff
@@ -295,3 +295,67 @@ func Shuffling(
 	// Split the shuffled list into epoch_length * committees_per_slot pieces.
 	return utils.SplitIndices(shuffledIndices, committeesPerEpoch), nil
 }
+
+// AttestationParticipants returns the attesting participants indices.
+//
+// Spec pseudocode definition:
+//   def get_attestation_participants(state: BeaconState,
+//     attestation_data: AttestationData,
+//     bitfield: bytes) -> List[ValidatorIndex]:
+//     """
+//     Returns the participant indices at for the ``attestation_data`` and ``bitfield``.
+//     """
+//     # Find the committee in the list with the desired shard
+//     crosslink_committees = get_crosslink_committees_at_slot(state, attestation_data.slot)
+//
+//	   assert attestation_data.shard in [shard for _, shard in crosslink_committees]
+//     crosslink_committee = [committee for committee,
```

Comment:
`committee for committee` ?

Reply:
it's on funky looking statement from the spec
`crosslink_committee = [committee for committee, shard in crosslink_committees if shard == attestation_data.shard][0]`